releasenotes/notes/fix-ping-config-f4339de3a55bff87.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix issue where for ping init config took priority over instance config
```

```suggestion
    Fix issue where init config for ping took priority over instance config.
```
Hopefully this hasn't changed the meaning. It was a little confusing as written

---

It hasn't changed the meaning, it's definitely more clear with the edit.